my_code.py

```diff
@@ -0,0 +1,33 @@
+import numpy as np
+import matplotlib
+import os
+from astropy.io import fits
+import glob
+data = fits.getdata('/Users/bostroem/Desktop/images/hubble_img.fits')
+col1 = data[20, :] 
+col2 = data[200, :]
+col3 = data[800, :]
+pyplot.figure()
+pyplot.plot(np.arange(1024), col1)
```

Comment:
Rather than defining the label names in the `pyplot.legend()` call, you could alternatively define `label="name"` in the plot commands and let `pyplot.legend()` automatically determine the mapping from label to style. For example:

```python
plt.plot(..., label='Column 20')
plt.axhline(..., label='Mean')
plt.legend()
```